mlshvch/hometask-6/scrapper.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'csv'
+require 'capybara'
+require 'capybara/dsl'
+require 'selenium-webdriver'
+
+SCRAPING_WEBSITE = 'https://www.onliner.by'
+NEWS_CSS_SELECTORS = %w[widget-1-1 widget-2-1 widget-3-1 widget-4-1 widget-5-1 widget-6-1 widget-7-1 widget-8-1
+                        widget-9-1 widget-10-1 widget-11-1 widget-12-1].freeze
+CATALOG_CSS_SELECTOR = 'div.catalog-offers__item'
+PEOPLE_NEWS_SELECTOR = 'ul.b-opinions-main-2'
+# class for parsing webpage data
+class Scraper
+  include Capybara::DSL
+
+  attr_reader :result
+
+  def initialize
+    @result = []
+    configure_capybara
+  end
+
+  def scrape_data
+    extract_html if @html.nil?
+    extract_news_data
+    extract_catalog_data
+    write_to_csv
+  end
+
+  private
+
+  def configure_capybara
+    Capybara.register_driver(:selenium) { |app| Capybara::Selenium::Driver.new(app, browser: :firefox) }
+    Capybara.javascript_driver = :chrome
```

Comment:
According to the Capybara docs, there is no such thing `:chrome` driver. Did you mean `:selenium_chrome`?